Yoast/javascript
# Convert LinkSuggestions sass to styled components

## Summary
* Converted the linksuggestions sass file to style components.
* The sass file has been removed

## Relevant technical choices:
* I've added an example of the linksuggestions.

## Test instructions

This PR can be tested by following these steps:

*

I made a pull for wordpress-seo-premium as well, see: https://github.com/Yoast/wordpress-seo-premium/pull/2312

## UI changes

## Quality assurance


Fixes #145
